lib/uris.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const querystring = require('querystring')
+
+const { kebabCase, omitBy } = require('lodash')
+
+// Services
+const ABILITIES = 'abilities'
+const STATE_HANDLER_PATH = 'state'
+const ACTION_HANDLER_PATH = 'action'
+const EVENT_HANDLER_PATH = 'event'
+
+// Controller
+const AUTH = 'auth'
+const SERVICES = 'services'
+const USERS = 'users'
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Correct this attempt to modify "ABILITIES" or use "let" in its declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3500)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "ABILITIES" as this name is already used in declaration at line 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2814)